<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>playingField</class>
 <widget class="QDialog" name="playingField">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="field1_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="field1_3">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>180</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="field3_1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="field3_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="field3_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="field2_1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="field2_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="field2_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>280</y>
     <width>160</width>
     <height>175</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="field1_1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
